api/app.js

```diff
@@ -68,7 +68,21 @@ module.exports = function(settings, callback) {
         ca: fs.readFileSync(settings.publicTls.caCertFile)
     }
   }
-
+  var checkBinding = function(req, res, next){
+    serviceBrokerUtil.checkBinding(req.params,function(error,result){
+      if(error){
+        res.status(HttpStatus.INTERNAL_SERVER_ERROR).send({});
+      }else{
+        if(result.statusCode == HttpStatus.OK){
+          next();
+        }else if(result.statusCode == HttpStatus.NOT_FOUND){
+          res.status(HttpStatus.NOT_FOUND).send({"message": "binding_info_not_found"});
```

Comment:
Suggest to return 403 forbidden with msg "The request cannot be performed without an existing service binding"